wolfssl/wolfcrypt/types.h

```diff
@@ -112,6 +117,13 @@
             #define WOLFCRYPT_SLOW_WORD64
         #endif
     #endif
+
+#elif defined(WC_16BIT_CPU)
+        #undef WORD64_AVAILABLE
+        typedef unsigned short wolfssl_word;
```

Comment:
wolfssl_word should be a wordXX type like all other platforms, also, unsigned short isn't used in WC_16BIT_CPU which makes this especially confusing.